scripts/create-clusters.sh

```diff
@@ -25,7 +25,7 @@ set -o pipefail
 source "${BASH_SOURCE%/*}/util.sh"
 NUM_CLUSTERS="${NUM_CLUSTERS:-2}"
 KIND_IMAGE="${KIND_IMAGE:-}"
-KIND_TAG="${KIND_TAG:-}"
+KIND_TAG="${KIND_TAG:-v1.18.4}"
```

Comment:
In the [pre-commit hook](https://github.com/kubernetes-sigs/kubefed/blob/ecd49000b586e2ed29e3a1b938e2f85772358843/scripts/pre-commit.sh#L139) we're using `v1.19.1@sha256:98cf5288864662e37115e362b23e4369c8c4a408f99cbc06e58ac30ddc721600`. I'd recommend to use the same version by default or rather pull the version in from there or some other common location so we don't have to maintain two locations for this variable.

Reply:
That makes sense. I referred an old version before 1.19.1. It is updated. Thanks @makkes